content/docs/testing-recipes.md

```diff
@@ -28,42 +29,24 @@ We can write a test for this component:
 
 ```jsx
 // hello.test.js
+import "@testing-library/react/cleanup-after-each";
+import "@testing-library/jest-dom/extend-expect";
+import { render } from "@testing-library/react";
 import Hello from "./hello";
-import { render } from "react-dom";
-import { act } from "react-dom/test-utils";
-
-let container = null
-beforeEach(() => {
-  // setup a DOM element as a render target
-  container = document.createElement('div')
-  // attach the container to document so events works correctly
-  document.body.appendChild(container)
-})
-
-afterEach(() => {
-  // cleanup on exiting
-  container.remove();
-  container = null  
-})
-
-it("renders with or without a name", () => {
-  act(() => {
-    render(<Hello />, container);
-  });
-  expect(container.textContent).toBe("Hey, stranger");
 
-  act(() => {
-    render(<Hello name="Sophie" />, container);
-  });
-  expect(container.textContent).toBe("Hello Sophie!");
+test("renders without a name", () => {
+  const { container } = render(<Hello />);
+  expect(container).toHaveTextContent("Hey, stranger");
+});
 
-  act(() => {
-    render(<Hello name="Flarnie" />, container);
```

Comment:
Having this additional test gives us no more confidence, so I don't have it in my version.